### DISCUSSION TITLE
click behaviour on dropdown multiselect

### DISCUSSION THREAD

**Discussion by augustobr1to** [zendeskgarden/react-components#1401] [Q&A]
I'm looking for a way to select an item in [multiselect component](https://garden.zendesk.com/components/multiselect) without closing menu (just close when click outside), there are an easiest way to do this?

**Comment by jzempel** [comment]
Please use Garden's new [Combobox](https://garden.zendesk.com/components/combobox#multiselect). It has the behavior you want along with much-improved accessibility overall.
